File: choitaejin/week7/boj_1789.js
const readline = require('readline');

const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout,
});

let input;
let cnt = 0;

rl.on('line', (line) => {
  
  input = parseInt(line);
  let i = 1;
  let sum = 0;

  while (sum <= input) {
    sum += i;
    cnt++;
    i++;
  }

  if (sum > input) cnt--;

  rl.close();

}).on('close', () => {
  console.log(cnt);
  process.exit();
});

File: ojuno/week7/boj_1789.cpp
#include <iostream>
using namespace std;

int main(){
    int cnt = 0;
    long long s;
    long long sum = 0;
    int next = 1;
    
    cin >> s;

    while(sum + next <= s){
        sum += next;
        next++;
        cnt++;
    }
    
    cout << cnt;
    
    return 0;
}